db.js

```diff
@@ -671,7 +681,7 @@ exports.init = function (sbot, config) {
         if (idx.indexesContent()) idx.processRecord(record, seq, pValue)
       }
 
-      cb(null, record.offset)
+      cb()
```

Comment:
The result seemed like it was never used?

Reply:
Right